book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rentals(rental)
+    @rental.push(rental)
+    rental.belong_to_book = self
+  end
```

Comment:
- [ ] The method should allow users to create a rental given a person and date. That's all. The rental `initialize` method will add the rental to the respective arrays.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```